docs/disk_failure.md

```diff
@@ -0,0 +1,48 @@
+# Disk failure
+
+The `diskFailure` field offers a way to apply a disk failure on a specific process and path.
+
+## How it works
+
+The disruption runs an eBPF disk failure program used to intercept system calls and inject errors. 
+It is used to prevent certain processes from accessing certain files. 
+It defines a target process and a filter path, and if the process is trying to 
+open a file that matches the filter path, an -ENOENT error will be injected, 
+preventing the process from opening the file.
+The source code of the eBPF disk failure program is [here](../ebpf/disk-failure). 
+
+### Notes
+
+:warning: If you target a node with a `"/"` filter path, the disruption will catch all openat syscalls of the node.
+If it's the case, your last chance is to restart the node manually because you will not be able to connect remotely to 
+it and do any command line.
+
+### Known issues
+
+Because an eBPF program has a limited memory and you cannot do dynamic loop, the filter path could not exceed `62` characters.
```

Comment:
We don't check for this character limit in the `Validate()` function of the Disk Failure spec